src/v2/Apps/Show/ShowApp.tsx

```diff
@@ -52,11 +56,17 @@ export const ShowApp: React.FC<ShowAppProps> = ({ show }) => {
             contextPageOwnerType,
           }}
         >
-          <Box mt={4} mb={[4, 6]}>
-            <ShowInstallShots show={show} mt={4} mb={6} />
-          </Box>
+          {arrivedFromFair && (
+            <ShowNavigationBanner show={show} mt={2} mb={4} />
+          )}
+
+          {show?.images!.length > 0 && (
```

Comment:
#minor is there a reason we're using `images!` instead of `images?`? I'm not sure if it's possible for a show to have no associated images, but I think this would crash if so

Reply:
"!" using to avoid null_check error. We always get array (empty or with items). If we use "?" we should cast null/unndefined types to number to bypass linter error